### ISSUE TITLE
Inference ERROR: Not compiled with GPU support！

### ISSUE THREAD

**Issue by a2658927017** [yerfor/Real3DPortrait#35]
To create a public link, set `share=True` in `launch()`.
Changes of a2m_ckpt detected, reloading model
Changes of head_ckpt detected, reloading model
Changes of torso_ckpt detected, reloading model
infer inputs : {'src_image_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\c95821a5e99f7a174002c9b9ce43b7e8c27b35ff\\Macron.png', 'drv_audio_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\10e38d2a0e8854b09cf1b695e2305e3219e729af\\Obama_5s.wav', 'drv_pose_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\\May_5s.mp4', 'bg_image_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\77e40a84169eb20cd5632ed343840895826c8a11\\bg.png', 'blink_mode': 'period', 'temperature': 0.2, 'mouth_amp': 0.45, 'out_mode': 'concat_debug', 'map_to_init_pose': True, 'low_memory_usage': False, 'hold_eye_opened': False, 'a2m_ckpt': 'D:\\Real3DPortrait\\checkpoints\\240210_real3dportrait_orig\\audio2secc_vae\\model_ckpt_steps_400000.ckpt', 'head_ckpt': None, 'torso_ckpt': 'D:\\Real3DPortrait\\checkpoints\\240210_real3dportrait_orig\\secc2plane_torso_orig\\model_ckpt_steps_100000.ckpt', 'min_face_area_percent': 0.2, 'out_name': '', 'seed': 42}
| WARN: egs/egs_bases/audio2motion/vae.yaml not exist.
| WARN: D:\Real3DPortrait\checkpoints\240210_real3dportrait_orig\th1kh_512_audio2motion\base.yaml not exist.
| load 'model' from 'D:\Real3DPortrait\checkpoints\240210_real3dportrait_orig\audio2secc_vae\model_ckpt_steps_400000.ckpt', strict=True
| WARN: D:\Real3DPortrait\checkpoints\240210_real3dportrait_orig\secc2plane_torso_orig\secc_img2plane_orig.yaml not exist.
| load 'model' from 'D:\Real3DPortrait\checkpoints\240210_real3dportrait_orig\secc2plane_torso_orig\model_ckpt_steps_100000.ckpt', strict=True
| Warning: Assigned --torso_ckpt which also contains head, but --head_ckpt is also assigned, skipping the --head_ckpt.
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
W0000 00:00:1709793788.496335  170060 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
face area percent 0.22489264198096937 larger than threshold 0.2, directly use the input image...
'cp' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
Extracted wav file (16khz) from C:\Users\yanfa-06\AppData\Local\Temp\gradio\10e38d2a0e8854b09cf1b695e2305e3219e729af\Obama_5s.wav to C:\Users\yanfa-06\AppData\Local\Temp\gradio\10e38d2a0e8854b09cf1b695e2305e3219e729af\Obama_5s_16k.wav.
Loading the HuBERT Model...
Loading the Wav2Vec2 Processor...
W0000 00:00:1709793794.119602  170060 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
batch 0 | loss_lan: 0.53, loss_reg_id: 0.64,loss_reg_exp: 1.22
| To extract pose from C:\Users\yanfa-06\AppData\Local\Temp\gradio\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\May_5s.mp4
lms_2d file not found, try to extract it from video... C:\Users\yanfa-06\AppData\Local\Temp\gradio\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\May_5s_lms.npy
W0000 00:00:1709793795.607723  170060 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
W0000 00:00:1709793795.624351  170060 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
| Extracted pose from C:\Users\yanfa-06\AppData\Local\Temp\gradio\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\May_5s.mp4
{'src_image_name': 'infer_out/tmp/cropped_src_img.png', 'drv_audio_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\10e38d2a0e8854b09cf1b695e2305e3219e729af\\Obama_5s.wav', 'drv_pose_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\\May_5s.mp4', 'bg_image_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\77e40a84169eb20cd5632ed343840895826c8a11\\bg.png', 'blink_mode': 'period', 'temperature': 0.2, 'mouth_amp': 0.45, 'out_mode': 'concat_debug', 'map_to_init_pose': True, 'low_memory_usage': False, 'hold_eye_opened': False, 'a2m_ckpt': 'D:\\Real3DPortrait\\checkpoints\\240210_real3dportrait_orig\\audio2secc_vae\\model_ckpt_steps_400000.ckpt', 'head_ckpt': None, 'torso_ckpt': 'D:\\Real3DPortrait\\checkpoints\\240210_real3dportrait_orig\\secc2plane_torso_orig\\model_ckpt_steps_100000.ckpt', 'min_face_area_percent': 0.2, 'out_name': '', 'seed': 42}
| audio-to-motion finished
rendering drv secc:   0%|                                                                                                                         | 0/3 [00:00<?, ?it/s]create rasterizer on device cuda:0
rendering drv secc:   0%|                                                                                                                         | 0/3 [00:00<?, ?it/s]
Inference ERROR: Not compiled with GPU support
Failed to generate

![803H5$XI3X7MFXGRETRHL4W](https://github.com/yerfor/Real3DPortrait/assets/105424238/3e210252-9cb1-44ba-9016-ffd26c6f1afa)

What am I supposed to do。

**Comment by a2658927017** [comment]
Extracted wav file (16khz) from C:\Users\yanfa-06\AppData\Local\Temp\gradio\10e38d2a0e8854b09cf1b695e2305e3219e729af\Obama_5s.wav to C:\Users\yanfa-06\AppData\Local\Temp\gradio\10e38d2a0e8854b09cf1b695e2305e3219e729af\Obama_5s_16k.wav.
W0000 00:00:1709796163.188500  158496 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
batch 0 | loss_lan: 0.53, loss_reg_id: 0.64,loss_reg_exp: 1.22
| To extract pose from C:\Users\yanfa-06\AppData\Local\Temp\gradio\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\May_5s.mp4
lms_2d file not found, try to extract it from video... C:\Users\yanfa-06\AppData\Local\Temp\gradio\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\May_5s_lms.npy
W0000 00:00:1709796164.893587  158496 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
W0000 00:00:1709796164.914156  158496 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
| Extracted pose from C:\Users\yanfa-06\AppData\Local\Temp\gradio\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\May_5s.mp4
{'src_image_name': 'infer_out/tmp/cropped_src_img.png', 'drv_audio_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\10e38d2a0e8854b09cf1b695e2305e3219e729af\\Obama_5s.wav', 'drv_pose_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\ea1ab761e00e431654ac8293a6d1e93a2a4ca7ef\\May_5s.mp4', 'bg_image_name': 'C:\\Users\\yanfa-06\\AppData\\Local\\Temp\\gradio\\77e40a84169eb20cd5632ed343840895826c8a11\\bg.png', 'blink_mode': 'period', 'temperature': 0.2, 'mouth_amp': 0.45, 'out_mode': 'concat_debug', 'map_to_init_pose': True, 'low_memory_usage': False, 'hold_eye_opened': False, 'a2m_ckpt': 'D:\\Real3DPortrait\\checkpoints\\240210_real3dportrait_orig\\audio2secc_vae\\model_ckpt_steps_400000.ckpt', 'head_ckpt': None, 'torso_ckpt': 'D:\\Real3DPortrait\\checkpoints\\240210_real3dportrait_orig\\secc2plane_torso_orig\\model_ckpt_steps_100000.ckpt', 'min_face_area_percent': 0.2, 'out_name': '', 'seed': 42}
| audio-to-motion finished
rendering drv secc: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:01<00:00,  1.68it/s]
blinking secc: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 128/128 [00:00<00:00, 618.27it/s]
Real3D-Portrait is rendering frames:   0%|                                                                                                      | 0/128 [00:00<?, ?it/s]Setting up PyTorch plugin "bias_act_plugin"... Failed!
Real3D-Portrait is rendering frames:   0%|                                                                                                      | 0/128 [00:00<?, ?it/s]
Inference ERROR: DLL load failed while importing bias_act_plugin: 找不到指定的模块。
Failed to generate

**Comment by LLlio** [comment]
@a2658927017 你好，我也碰到你发的第二个报错了，你解决了吗，怎么解决的
